rpa-manager/modules/ROOT/pages/processautomation-prepare-project-application.adoc

```diff
@@ -0,0 +1,111 @@
+= Manage Applications
+
+Process applications are third-party application installations that are required for the execution of a process. 
```

Comment:
"to execute a process"?

Reply:
This surly can be worded better.
The content refers to a business process we want to automate. A business process consists of a series of steps a human executes to reach a certain goal.
For example:
1. Open an email with the application Microsoft Outlook.
2. Download and open the attachment.
3. Copy the account number, name, and amount to a data sheet with the application Microsoft Excel and save it.
To automate this business process the bot needs to access the applications Outlook and Excel.
In the Process Applications view we list all available applications and where to find them.
In the Bots view we assign those applications to bots that can access them.
In the Process Management view we assign the required application to the project.
Only bots with access to the 'required' applications can execute the process.

Reply:
How about:
Process applications are third-party application installations that business experts use to execute the process.
?

Reply:
I like it!